modules/installation-network-user-infra.adoc

```diff
@@ -16,6 +16,12 @@ It is recommended to use the DHCP server to manage the machines for the cluster
 long-term. Ensure that the DHCP server is configured to provide persistent IP
 addresses and host names to the cluster machines.
 
+The Kubernetes API server must be able to resolve the node names of the cluster
+machines. If the API servers and worker nodes are in different zones, you can
+configure a default DNS search zone to allow the API server to resolve the
+node names. The API server can also resolve the host names if you configure each
+host to record its fully-qualified domain name in its node object. 
```

Comment:
Instead of "The API server can also resolve the host names..." I might consider something like, "Another acceptable approach is simply to always refer to hosts by their fully-qualified domain name in both the node object and all DNS requests".